docs/accessibility.rst

```diff
@@ -0,0 +1,182 @@
+Accessibility Testing
+======================
+
+The bok-choy framework includes the ability to test web pages for accessibility
+using `Google Accessibility Developer Tools`_.
+
+The audit rules that are checked and optionally the scope of the audit
+within the webpage itself are defined on each Page Object's definition.
+
+The general methodology for enabling accessibility auditing is:
+
+* Define the accessibility rules to check for a page
+* (optional) Define the scope for accessibility auditing of a page
+* Trigger the audits either:
+
+  * Actively, by calling the Page Object class's do_axs_audit method and then
+  asserting on the results returned.
+  * Passively, by setting the VERIFY_ACCESSIBILITY environment variable to True.
+  This will trigger an accessibility audit whenever a Page Object's wait_for_page
+  method is called, and raise an AccessibilityError if errors are found on the page.
+
+*Note that accessibility auditing is only supported with PhantomJS as the browser.*
+
+Define the accessibility rules to check for a page
+--------------------------------------------------
+
+A page object's list of accessibility rules to audit for that page are defined
+in the axs_audit_rules method. The code for the available rules can be found
```

Comment:
I was confused about what  'code' means here.   The javascript code has code values for each rule in it. (e.g. BadAriaAttributeValue.js has `code: 'AX_ARIA_04'` in the addRule call) 

I was then confused why ''badAriaAttributeValue'' was being used in your example below.    It might be helpful to specify "javascript code" here, and to mention that the important part to look for is the 'name' attribute used in addRule.